### DISCUSSION TITLE
[Mapping] Le mapping Twitch News n'existe pas.

### DISCUSSION THREAD

**Discussion by DarkEag** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#13] [Q&A] [answered]
J'ai l'impression qu'il manque le mapping twitch news je ne le vois pas et non plus dans les dossiers ^^

**Comment by joelwurtz** [comment, marked as answer]
Nous n'avons pas les droit de diffusion pour ce mapping, il n'est pas donc pas disponible sur ce repo
